### PR TITLE
[IMP] mass_mailing: Improve list view

### PR DESCRIPTION
**Pull request by kig-odoo**
Task : https://www.odoo.com/web#id=42548&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.d9ad5aff57c4bc4d769a700fffbfbf72

### CODE REVIEW

File: addons/mass_mailing/views/mass_mailing_views.xml [hunk 1]
```diff
@@ -394,11 +394,10 @@
...
                 <tree string="Mailings">
-                    <field name="name"/>
-                    <field name="sent"/>
-                    <field name="delivered"/>
-                    <field name="opened"/>
-                    <field name="replied"/>
+                    <field name="name" string="Subject"/>
+                    <field name="email_from" />
+                    <field name="mailing_model_id" string="Recipients"/>
+                    <field name="bounced_ratio"/>
```
Comment (chienandalu) [inline addons/mass_mailing/views/mass_mailing_views.xml]: Maybe a progress bar?

File: addons/mass_mailing/views/mass_mailing_views.xml [hunk 2]
```diff
@@ -394,11 +394,10 @@
             <field name="priority">10</field>
             <field name="arch" type="xml">
                 <tree string="Mailings">
-                    <field name="name"/>
-                    <field name="sent"/>
-                    <field name="delivered"/>
-                    <field name="opened"/>
-                    <field name="replied"/>
+                    <field name="name" string="Subject"/>
```
Comment (chienandalu) [inline addons/mass_mailing/views/mass_mailing_views.xml]: Why not sent_date?

File: addons/mass_mailing/views/mass_mailing_views.xml [hunk 3]
```diff
@@ -394,11 +394,10 @@
             <field name="priority">10</field>
             <field name="arch" type="xml">
                 <tree string="Mailings">
-                    <field name="name"/>
-                    <field name="sent"/>
-                    <field name="delivered"/>
-                    <field name="opened"/>
-                    <field name="replied"/>
```
Comment (chienandalu) [inline addons/mass_mailing/views/mass_mailing_views.xml]: Isn't it interesting to keep these?

**Comment by kig-odoo** [comment]
Merged at https://github.com/odoo/odoo/commit/ec884790485b34ef4c6f66e93bf8b21e366676bf

**Pull request closed without merging**